File: 1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int s, mx = -1;
        for(int i = 0 ; i < accounts.length ; i++){
                s = 0;
            for(int j = 0 ; j < accounts[i].length ; j++){
                s += accounts[i][j];
                mx = Math.max(s,mx);
            }
        }
        return mx;
    }
}

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.kt
class Solution {
    fun maximumWealth(accounts: Array<IntArray>): Int {
        var mx = Int.MIN_VALUE
        for(i in accounts.indices){
           var  s = 0
            for(j in accounts[i].indices){
                s += accounts[i][j]
                mx = maxOf(s,mx)
            }
        }
        return mx
    }
}